GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -3,21 +3,26 @@ import UIKit
 class ChatView: EngagementView {
     let tableView = UITableView()
     let messageEntryView: ChatMessageEntryView
+    let newMessageIndicatorView: NewMessageIndicatorView
     var numberOfSections: (() -> Int?)?
     var numberOfRows: ((Int) -> Int?)?
     var itemForRow: ((Int, Int) -> ChatItem?)?
     var fileTapped: ((LocalFile) -> Void)?
     var downloadTapped: ((FileDownload) -> Void)?
     var callBubbleTapped: (() -> Void)?
     var choiceOptionSelected: ((ChatChoiceCardOption, String) -> Void)!
+    var newMessageIndicatorTapped: (() -> Void)?
+    var chatScrolledToTheEnd: ((Bool) -> Void)?
```

Comment:
`chatScrolledToBottom` or even just `scrolledToBottom` since this is already the `ChatView`?

Reply:
Renamed as per last suggestion